### DISCUSSION TITLE
FAQ

### DISCUSSION THREAD

**Discussion by itprojects** [johnfactotum/foliate#640] [Q&A] [answered]
Frequently Asked Questions & Answers Thread.

Do we need one?

**Comment by johnfactotum** [comment, marked as answer]
Why would we need one?

**Reply by itprojects** [reply]
Because it reduces the number of entry-level questions in the Issues category? There will only be more of those.

**Reply by johnfactotum** [reply]
There aren't really many of those. Anyway, this is already marked as wontfix, per https://github.com/johnfactotum/foliate/issues/133.
